modules/persistent-storage-csi-migration-overview.adoc

```diff
@@ -6,27 +6,6 @@
 [id="persistent-storage-csi-migration-overview_{context}"]
 = Overview
 
-With the automatic migration feature enabled, volumes provisioned using in-tree storage plug-ins supported by this feature are migrated to their counterpart CSI drivers.
+Volumes provisioned using in-tree storage plug-ins supported by this feature are migrated to their counterpart Container Storage Interface (CSI) drivers. This process does not involve any data migration, {product-title} only translates the persistent volume object in memory. As a result, the translated persistent volume object is not stored on disk, nor is its contents changed.
```

Comment:
Per the ISG:
> ! using noun
To avoid ambiguity, replace this gerund with either "by using" or "that uses".

I also suggest simplifying the last sentence.

```suggestion
Volumes provisioned by using in-tree storage plug-ins supported by this feature are migrated to their counterpart Container Storage Interface (CSI) drivers. This process does not involve any data migration. {product-title} only translates the persistent volume object in memory. As a result, the translated persistent volume object is not stored on disk, and its contents are not changed.
```